cds_ils/importer/documents/importer.py

```diff
@@ -266,7 +284,8 @@ def search_for_matching_documents(self):
                 document = document_class.get_record_by_pid(match)
                 if document.get("edition") != self.json_data.get("edition"):
                     matches.remove(match)
-        return matches
+
+        return self.validate_found_matches(matches)
```

Comment:
wouldn't be easier to use this function in the _match document instead of changing the returned type here ? 
what if you want to re-use this function without the validation ?

Reply:
good one actually, i was hesitating where to apply it